modules/persistent-storage-using-a-shared-configmap-in-a-pod.adoc

```diff
@@ -0,0 +1,51 @@
+:_content-type: PROCEDURE
+
+[id="persistent-storage-using-a-shared-configmap-in-a-pod_{context}"]
+= Using a shared configMap in a pod
+
+[role="_abstract"]
+
+.Prerequisites
+
+* You created a `SharedConfigMap` object for the config map that you want to share across namespaces in the cluster.
+* You have privileges to perform the following actions:
+  * Create pods and access instances of available `sharedconfigmaps.sharedresource.openshift.io`. In other words, you can run `oc get sharedconfigmaps` and get a non-empty list back.
+  * Determine if any of the service accounts available to you in your namespace are allowed to use a given `SharedConfigMap` resource. In other words, you can run `oc adm policy who-can use <identifier of specific SharedConfigMap>` to see if a service account in your namespace is listed.
+  * Establish, or ask someone to establish, the necessary role-based access control (RBAC) so that you can discover `SharedConfigMaps` resources and allow service accounts to use `SharedConfigMaps` resources.
+
+.Procedure
+
+. Grant a given `ServiceAccount` RBAC permissions to use the `SharedConfigMap` resource in its pod:
++
+[source,terminal]
+----
+oc create role shared-resource-my-share --verb=use --resource=sharedconfigmaps.sharedresource.openshift.io --resource-name=my-share
+oc create rolebinding shared-resource-my-share --role=shared-resource-my-share --serviceaccount=my-namespace:default
+----
+
+. Access the `SharedConfigMap` resource from a pod:
++
+[source,terminal]
+----
+$ oc apply -f - <<EOF
+kind: Pod
+apiVersion: v1
+metadata:
+  name: my-app
+  namespace: my-namespace
+spec:
+  serviceAccountName: default
+
+# containers omitted …. Follow standard use of ‘volumeMounts’ for referencing your shared resource volume
+
+
+    volumes:
+    - name: my-csi-volume
+      csi:
+        readOnly: true # editor note: eventually, we’ll have to add pointers to k8s atomic secret sync mechanism when we wire that up
```

Comment:
```suggestion
        readOnly: true 
```

Reply:
Updated here and in modules/persistent-storage-using-a-shared-secret-in-a-pod.adoc